phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -192,6 +191,14 @@ public static IndexDisableLoggingType fromValue(byte[] value) {
         }
     }
 
+    /**
+     * Which table to use as the source table
+     */
+    public enum SourceTable {
```

Comment:
nit: We have a similar enum in the IndexScrutinyTool. Can we re use it?

Reply:
It has a BOTH option which is not supported that is why I didn't want to use it.

Reply:
I agree with Gokcen, does not hurt to use it. In the future, we may want to use both. Would be a good extension.